### PR TITLE
Add "forced-use-1_17-resource-pack" option

### PR DESCRIPTION
**Pull request by LemonCaramel**
_(I'm sorry for opening the pull request so often. 😢)_

This option will be of great help for servers that require the use of server resource pack. _(FPS, mini game, etc...)_

### CODE REVIEW

File: common/src/main/java/us/myles/ViaVersion/api/ViaVersionConfig.java [hunk 1]
```diff
@@ -366,4 +366,11 @@
      * @return true if enabled
      */
     boolean isIgnoreLong1_16ChannelNames();
+
+    /**
+     * Force the 1.17+ client to use server resource pack.
```
Comment (KennyTV) [inline common/src/main/java/us/myles/ViaVersion/api/ViaVersionConfig.java:371]:
```suggestion
     * Force 1.17+ client to accept the server resource pack.
```

File: common/src/main/resources/assets/viaversion/config.yml [hunk 2]
```diff
@@ -151,6 +151,9 @@ vine-climb-fix: false
 # CraftBukkit had this limit hardcoded until 1.16, so we have to assume any server/proxy might have this arbitrary check present.
 ignore-long-1_16-channel-names: true
 #
+# Force the 1.17+ client to use server resource pack.
```
Comment (KennyTV) [inline common/src/main/resources/assets/viaversion/config.yml:154]:
```suggestion
# Force 1.17+ client to accept the server resource pack; they will automatically disconnect if they decline.
```

**Review by KennyTV — APPROVED** [review]
No worries, thank you for the pr, just some minor text change suggestions 😛

**Comment by LemonCaramel** [comment]
I think I pressed something wrong, is it right that I did it right?

**Comment by KennyTV** [comment]
That worked, thanks!

**Pull request merged**